### ISSUE TITLE
NTR: Modifier: testicle elevation relieves pain  (General history, localization & modifiers)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7857]
New term request Modifier: testicle elevation relieves pain  (brumme_220503150954): Prehn sign

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000170
- label: Pain alleviated by testicle elevation
- parents: [Ameliorated by (HP:0025254)](https://hpo.jax.org/app/browse/term/HP:0025254)
- pmid: PMID:19378875
- def: This refers to testicular pain that is lessened or made more bearable by elevation of the testicle.
- comment: Prehn sign, the relief of pain with the elevation of the testis, may be elicited in patients with epididymitis, although this is not a reliable finding.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7857
- synonyms: Testicle elevation relieves pain(NONE)|Prehn's sign(NONE)
